.github/workflows/_linux_accelerate.yml

```diff
@@ -0,0 +1,156 @@
+name: Linux Accelerate Test
+
+on:
+  pull_request:
+    branches:
+      - main
+    paths:
+      - '.github/scripts/spec.py'
+      - '.github/actions/print-environment/action.yml'
+      - '.github/workflows/_linux_accelerate.yml'
+  workflow_dispatch:
+    inputs:
+      pytorch:
+        required: false
+        type: string
+        default: 'nightly'
+        description: Pytorch branch/commit
+      python:
+        required: false
+        type: string
+        default: '3.10'
+        description: Python version
+      runner:
+        required: true
+        type: string
+        default: 'linux.idc.xpu'
+        description: Runner label
+      nightly_whl:
+        required: false
+        type: string
+        default: ''
+        description: Pytorch nightly wheel version
+      accelerate:
```

Comment:
Do we need add a parameter for transformers version?

Reply:
That's a good question. These 2 projects have circle dependencies. My understanding is:
1. Transformers depend on Accelerate library for basic functionality
2. Accelerate library either does not depend on Transformers library or at least does not depend for basic features
3. Accelerate uses Transformers in tests
4. Accelerate uses Transformers in some utilities. Mostly for tests, but without a deep dive I am not 100% sure that there is no usage in Accelerate for some library features.

When we install Accelerate for testing we also get Transformers installed. As far as I see it will install latest Transformers version.

Overall, I think it will be wise to pin Transformers version for better test stability. I will add a change today.

Reply:
Added pin for Transformers version.